vcd/config_test.go

```diff
@@ -832,3 +880,27 @@ func importStateIdVmObject(orgName, vdcName, vappName, vmName, objectIdentifier
 			objectIdentifier, nil
 	}
 }
+
+func setBoolFlag(variable *bool, name, envVar, help string) {
```

Comment:
You know, same as in govcd PR, for me personally this function is confusing, mostly because flag.BoolVar definition in its own is not user friendly. I'd appreciate a comment in the file.

Reply:
Added comment. Flag operations are not easy to grasp at first.